modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/FieldsTable.js

```diff
@@ -90,7 +127,10 @@ function FieldsTable({portletNamespace}) {
 
 					<ClayTable.Body>
 						{fields.map((field) => {
-							const included = selectedFields.includes(field);
+							const included = !!selectedFields.find(
```

Comment:
if you need a boolean you can use `Array.prototype.some()`

Reply:
that's great. I didn't know about that function

Reply:
Eheheh somebody should create an rss feed that keeps people updated every time there's a new array method coming in :P